# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: tools/lint/trojan-source.yml
# Commit: 3f167f8b3137
# Full Hash: 3f167f8b313749f50f135e189e5854bef7ac0c73
# Author: Erich Gubler <erichdongubler@gmail.com>
# Date: 2025-10-03 18:08 +0000
# Regressor Bug: 1991551
# File Overlap Count: 23
# Overlapping Files: third_party/rust/wgpu-hal/Cargo.toml, supply-chain/audits.toml, third_party/rust/naga/src/back/spv/writer.rs, third_party/rust/naga/src/back/msl/writer.rs, third_party/rust/wgpu-types/src/lib.rs
# Description:
#   Bug 1991551 - build(webgpu): update WGPU to 8c4aebc0c1b75e3f7412dfc180745a0717fc30ec r=webgpu-reviewers,supply-chain-reviewers,nical
# ==============================================================================

diff -r 823439f307a0 -r 3f167f8b3137 tools/lint/trojan-source.yml
--- a/tools/lint/trojan-source.yml	Fri Oct 03 18:01:30 2025 +0000
+++ b/tools/lint/trojan-source.yml	Fri Oct 03 18:08:29 2025 +0000
@@ -22,6 +22,7 @@
         - third_party/rust/icu_provider/src/hello_world.rs
         - third_party/rust/icu_segmenter/src/grapheme.rs
         - third_party/rust/icu_segmenter/src/line.rs
+        - third_party/rust/ron/src/util/span_substring.rs
         - third_party/rust/unicode-width/src/lib.rs
         - third_party/rust/unicode-width/tests/tests.rs
         - security/nss/gtests/mozpkix_gtest/pkixnames_tests.cpp
